src/main/java/com/emc/object/ObjectConfig.java

```diff
@@ -200,6 +207,9 @@ public SmartConfig toSmartConfig() {
             smartConfig.withProperty(prop, properties.get(prop));
         }
 
+        // CONNECT_TIMEOUT
+        smartConfig.setProperty(ClientConfig.PROPERTY_CONNECT_TIMEOUT, connectTimeout);
+
```

Comment:
also need to set read timeout here (if it is non-zero)

Reply:
done